gateway/src/apicast/policy/http_proxy/Readme.md

```diff
@@ -51,23 +53,25 @@ used.
     {
       "name": "apicast.policy.http_proxy",
       "configuration": {
-          "all_proxy": "http://192.168.15.103:8888/",
-          "https_proxy": "https://192.168.15.103:8888/",
-          "http_proxy": "https://192.168.15.103:8888/"
+          "all_proxy": "http://foo:bar@192.168.15.103:8888/",
+          "https_proxy": "https://foo:bar@192.168.15.103:8888/",
```

Comment:
proxy urls should all be `http://`, `https` not supported

Reply:
fixed